### PR TITLE
ECOSYS-47 Payara Micro Maven Plugin Testcases and CI integration

### PR DESCRIPTION
**Pull request by jGauravGupta**

### CODE REVIEW

File: payara-micro-maven-plugin/src/main/java/fish/payara/maven/plugins/micro/StartMojo.java [hunk 1]
```diff
@@ -158,8 +158,8 @@ public void execute() throws MojoExecutionException {
             int indice = 0;
             actualArgs.add(indice++, evaluateJavaPath());
 
-            if(!"false".equalsIgnoreCase(debug)) {
-                if(Boolean.parseBoolean(debug)) {
+            if (debug != null && !debug.equalsIgnoreCase("false")) {
+                if (Boolean.parseBoolean(debug)) {
```
Comment (pzygielo) [inline payara-micro-maven-plugin/src/main/java/fish/payara/maven/plugins/micro/StartMojo.java]: Why not keep `Boolean.parseBoolean` only, and drop outer conditions?

File: payara-micro-maven-plugin/src/main/java/fish/payara/maven/plugins/micro/StartMojo.java [hunk 2]
```diff
@@ -212,7 +212,7 @@ else if (option.getValue() != null) {
                             "your application tried to deploy twice: 1. as uber jar 2. as a separate war");
                 }
                 actualArgs.add(indice++, "--deploy");
-                if (exploded) {
+                if (Boolean.TRUE.equals(exploded)) {
```
Comment (pzygielo) [inline payara-micro-maven-plugin/src/main/java/fish/payara/maven/plugins/micro/StartMojo.java]: Why not use primitive at line 102?

File: payara-micro-maven-plugin/src/main/java/fish/payara/maven/plugins/micro/StartMojo.java (hunk 1 above)
Comment (jGauravGupta) [reply payara-micro-maven-plugin/src/main/java/fish/payara/maven/plugins/micro/StartMojo.java]: `debug` option can be used to enable/disable debug mode with default debug option  or customize debug option can be passed through it.

e.g:
`mvn payara-micro:start -Ddebug=true'
`mvn payara-micro:start -Ddebug=false'
`mvn payara-micro:start -Ddebug=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=9999'

where `Boolean.parseBoolean` only check for `true`.

Comment (pzygielo) [reply payara-micro-maven-plugin/src/main/java/fish/payara/maven/plugins/micro/StartMojo.java]: Right, I already looked at it in prev. PR and then I saw `add` called in l. 165. Two days - and I forgot that.
Thanks for checking.

**Pull request merged**